paddle/fluid/operators/hierarchical_sigmoid_op.h

```diff
@@ -28,20 +30,46 @@ template <typename T, int MajorType = Eigen::RowMajor,
 using EigenMatrix = framework::EigenMatrix<T, MajorType, IndexType>;
 using platform::Transform;
 
+std::vector<int64_t> cal_rows(const framework::LoDTensor* path) {
```

Comment:
```cpp
const framework::LoDTensor* path => const framework::LoDTensor& path
```

Reply:
https://google.github.io/styleguide/cppguide.html#Reference_Arguments

In fact it is a very strong convention in Google code that input arguments are values or const references while output arguments are pointers.

Reply:
done